src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppKeys.scala

```diff
@@ -260,10 +260,10 @@ trait SbtReactiveAppKeys {
   val minikube = InputKey[Unit]("minikube")
 
   /**
-   * If non-empty (default: "application.conf"), all rp-tooling.conf resources in dependencies will be prepended to the
-   * unmanaged application.conf file, or one will be created if none exists. To disable this behavior, specify "".
+   * If non-empty (default: "application.conf"), all resources with the given name in dependencies will be concatenated into an
+   * managed rp-application.conf file. To disable this behavior, specify "".
    */
-  val prependRpConf = SettingKey[String]("rp-prepend-rp-application-conf")
+  val prependRpConf = SettingKey[String]("rp-concat-to-rp-application-conf")
```

Comment:
Why was this changed to concat-to?

Reply:
Thanks. I hadn't noticed that the string parameter for a setting was actually used, unlike for tasks where it's a description and the val name is used.

Reply:
@patrick-premont Just to clarify: in sbt both `SettingKey` and `TaskKey` have a label string and a description string (more precisely they're defined in terms of an internal `AttributeKey` that takes a label and a description).

What's probably getting you confuses is that [`SettingKey.apply`](https://github.com/sbt/sbt/blob/v1.1.6/main-settings/src/main/scala/sbt/Structure.scala#L667-L673) and [`TaskKey.apply`](https://github.com/sbt/sbt/blob/v1.1.6/main-settings/src/main/scala/sbt/Structure.scala#L635-L641) take both label and description strings, but default the description to the empty string. On the other hand the [`settingKey` and `taskKey`](https://github.com/sbt/sbt/blob/v1.1.6/main/src/main/scala/sbt/BuildSyntax.scala#L15-L16) (and `inputKey`) macros instead **derive** the label from the `val` name and **require** the description.

This subtle difference has created some confusion, often with keys using their descriptions as their labels 😁.